### PR TITLE
Set min. PHP version in Composer config

### PR DESCRIPTION
**Pull request by szepeviktor**
Just that.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     }
   ],
   "require": {
+    "php": "^5.3",
```
Comment (kasparsd) [inline composer.json:14]: The platform PHP further down the file is set to `5.6.20` which matches the minimum version of WP core https://wordpress.org/about/requirements/

> WordPress also works with PHP 5.6.20+ and MySQL 5.0+, but these versions have reached official End Of Life and as such may expose your site to security vulnerabilities.

Should we set this to `>=5.6.20` then?

The goal is to support every PHP from 5.6.20 to 8.x.

Comment (szepeviktor) [reply composer.json:14]: I'm not familiar with this project.
https://github.com/xwp/stream/blob/1b308298417274b0030c0ecc6ceec5dbb33e27a2/stream.php#L34
This line contains 5.3

Please decide what to do!
A month ago my default moved to PHP 8.0

Comment (szepeviktor) [reply composer.json:14]: Friendly ping 🏓

**Comment by szepeviktor** [comment]
Done in 78d5369

**Pull request closed without merging**